committee-code-of-conduct/election.md

```diff
@@ -0,0 +1,76 @@
+# Kubernetes Code of Conduct Committee Elections
+
+This document outlines the process, for Code of Conduct Committee (CoCC) elections.
+
+### Length of Terms
+
+* The CoCC consists of 5 members. In the first election, the top 3 voted people will be appointed for 2 year terms and the other 2 members will be appointed for a 1 year term.
+* In alternating years either 2 or 3 seats will be up for election as terms expire.
+
+### Eligibility for voting
+
+* Voting will be done by members of the steering committee.
+
+### Eligibility for candidacy
+
+* Candidate commits to putting the interests of the community above the interests of their employer for all Code of Conduct Committee activities.
+* Is generally a responsible human
+* Does not have to be part of the Kubernetes or CNCF community
+* Characteristics which would be nice in any nominations:
+  * Previous experience on an Ethics Committee or Code of Conduct Committee is appreciated
+  * Has demonstrated integrity, professionalism, and positive influence within the community is appreciated
+  * Experience with the tools which we use to communicate (Zoom, Slack, GitHub, etc.) within the Kubernetes community is appreciated
+
+### Election process
+
+The steering committee will announce a request for nominations some time in July from any member of the following:
+* steering committee
+* sig chair
+* sig lead
+
+The committee will accept nominations for at least 3 weeks. After the announced nomination
+period the steering committee will hold a private ballot election using time-limited
+[Condorcet] ranking use the [Schulze method] to break ties. The steering committee
+will publicly announce the results of the election.
+
+### Maximal representation
+
+Since we require commitment that all members of the committee put the interests
+of the community above the interest of their employer while performing all duties
+related to the CoCC we do not have a maximal corperate representation limit.
+
+### Terms and Election Cycles
+
+Code of Conduct Committee members are elected to serve two year terms.
```

Comment:
This contradicts the 2/1 year term wording under the length of terms section above. What if this section was limited to just Election Cycles and we don't repeat ourselves on term length?

Reply:
fixed, I hope